SeleniumTests/LoginTests.cs

```diff
@@ -49,84 +49,90 @@ public void CloseBrowser()
         [TestMethod]
         public void AnInvalidUserName_WithAValidPassword()
         {
-            //** HINT **.   Manually navigate to http://iemosoft.com/selenium-test-login/ in a chrome browser
-            //              Right click the 'Username' INPUT control and select 'Inspect'
-            //              Note the id attribute (eg. <input id="userName">) 
-            //              Replace the string 'USER_NAME_ELEMENT_ID_HERE' with the id value you found in the html
-            _driver.FindElement(By.Id("USER_NAME_ELEMENT_ID_HERE")).SendKeys("aBadBadUserName");
-
-
-            //** NOTE: below, we are using the _driver object to ask the ChromeDriver to find a element on the pages with an Id of 'password'), 
-            //then we're going to send the value we want on that input control
-            _driver.FindElement(By.Id("password")).SendKeys("P@assword");
-
-            //** HINT **.   Next click the submit button here.
-            //              Manually navigate to http://iemosoft.com/selenium-test-login/ in a chrome browser
-            //              Right click the 'Submit' button and select 'Inspect'
-            //              Notice there is no Id attribute.
-            //              In the Elementents tab, <input type='button' onclick='onSubmit()' value='Submit'> should be highlighted, 
-            //              Right click on <input type="button"... and select Copy -> Copy XPath
-            //              Paste the xpath value in the submitButtonXPath variable below
-
-            var submitButtonXPath = ""; //**** PUT THE XPATH VALUE HERE!!!
-            _driver.FindElement(By.XPath(submitButtonXPath)).Click();
+            // does the test login
+            do_login("aBadBadUserName", "P@assword");
```

Comment:
The lowercase _ is a javascript convention.  In C# it would be DoLogin, or just Login();  I know this seems really trivial, but every shop will enforce it's conventions.

Reply:
Yeah, I actually don't think that's trivial. The lowercase_name is also a python thing, and that is were I have been spending a lot of time lately, but noted for the future.